app/Models/TrackFile.php

```diff
@@ -149,11 +155,21 @@ public function getDirectory()
 
     public function getFile()
     {
+        return "{$this->getDirectory()}/{$this->track_id}-v{$this->version}.{$this->extension}";
+    }
+
+    public function getUnversionedFile()
```

Comment:
This function, as well as `getUnversionedFilename`, are only needed to perform the filename migration, so they should be contained in the migration file rather than in the model class.